packages/slice-machine/lib/builders/SliceBuilder/index.tsx

```diff
@@ -110,6 +110,7 @@ const SliceBuilder: React.FunctionComponent = () => {
         // @ts-expect-error
         onSave={() => store.save(Model, setData)}
         isLoading={data.loading}
+        imageLoading={data.imageLoading}
```

Comment:
I understand why you can't change the shape of data, that would hard.
But please change Header parameter's name, we don't want to pass `imageLoading` everywhere.